go-controller/pkg/ovn/namespace.go

```diff
@@ -20,11 +20,13 @@ import (
 
 const (
 	// Annotation used to enable/disable multicast in the namespace
-	nsMulticastAnnotation        = "k8s.ovn.org/multicast-enabled"
-	routingExternalGWsAnnotation = "k8s.ovn.org/routing-external-gws"
-	routingNamespaceAnnotation   = "k8s.ovn.org/routing-namespaces"
-	routingNetworkAnnotation     = "k8s.ovn.org/routing-network"
-	bfdAnnotation                = "k8s.ovn.org/bfd-enabled"
+	nsMulticastAnnotation = "k8s.ovn.org/multicast-enabled"
+	// Annotations used by multiple external gateways feature
+	routingExternalGWsAnnotation     = "k8s.ovn.org/routing-external-gws"
+	routingNamespaceAnnotation       = "k8s.ovn.org/routing-namespaces"
+	routingNetworkAnnotation         = "k8s.ovn.org/routing-network"
+	bfdAnnotation                    = "k8s.ovn.org/bfd-enabled"
+	externalGatewayPodIPsAnnotations = "k8s.ovn.org/external-gw-podIPs"
```

Comment:
@tssurya let's keep these all lowercase...

Reply:
Arguably the other namespace annotations should go there too, but we can do that later.

Reply:
yeah let me move all these into types.go where they belong globally?